daily_contact_tracing_report.R

```diff
@@ -95,21 +95,33 @@ count_of_cases_for_all_time_by_person_type <- case_records %>%
   arrange(desc(n))
 
 isolated_cases <- case_records %>%
-  filter(cleared_for_campus != 'Y') %>% 
-  count(person_type, name = 'isolated') %>% 
-  arrange(desc(isolated))
+  mutate(isolation_status =
+           case_when(
+             cleared_for_campus != 'Y' |
+               (is.na(cleared_for_campus) & case_investigate == 1) ~ "currently_isolated",
+             TRUE ~ "previously_isolated"
+           )) %>% 
+  count(person_type, isolation_status) %>% 
+  pivot_wider(names_from = isolation_status, values_from = n) 
 
 quarantined_contacts <- contact_records %>%
-  filter(contact_released_from_quarantine == "0" | is.na(contact_released_from_quarantine)) %>%  
-  count(person_type, name = 'quarantined') %>% 
-  arrange(desc(quarantined))
+  mutate(
+    quarantine_status = case_when(
+      contact_released_from_quarantine == "0" |
+        is.na(contact_released_from_quarantine) ~ "currently_quarantined",
+      TRUE ~ "previously_quarantined"
+    )) %>% 
+  count(person_type, quarantine_status) %>% 
+  pivot_wider(names_from = quarantine_status, values_from = n) 
+  
 
 isolated_and_quarantined <- isolated_cases %>% 
-  full_join(quarantined_contacts, by = "person_type") %>%  
-  mutate_at(vars(isolated, quarantined),  ~replace(., is.na(.), 0)) %>% 
-  mutate(n = isolated + quarantined) %>% 
-  select(person_type, n) %>%
-  arrange(desc(n))
+  full_join(quarantined_contacts, by = "person_type") %>%   
+  mutate_at(vars(-person_type),  ~replace(., is.na(.), 0)) %>% 
+  mutate(n = currently_isolated + currently_quarantined,
+         anytime_n = previously_isolated + previously_quarantined) %>% 
+  select(person_type, n, anytime_n) %>%
```

Comment:
```suggestion
  mutate(n = currently_isolated + currently_quarantined,
         n_ever = currently_isolated + currently_quarantined + 
           previously_isolated + previously_quarantined) %>%
  select(person_type, n, n_ever) %>%
```

Jerne wanted the new variable to be the sum or past and present. I decided "ever" was a better label that "anytime"